src/exporter/wp_exporter.py

```diff
@@ -1271,15 +1271,34 @@ def display_report(self):
 
     def write_redirections(self):
         """
-        Update .htaccess file with redirections
+        Update .htaccess file with redirections.
+        The added redirections are for :
```

Comment:
Pas d'espace avant les ponctuations hautes en anglais

Reply:
Je t'avoue que je ne comprends pas ce que tu veux dire par là...

Reply:
"The added redirections are for:" sans espace entre "for" et les deux points